src/components/calcite-sortable-list/calcite-sortable-list.e2e.ts

```diff
@@ -13,10 +13,10 @@ describe("calcite-sortable-list", () => {
     let page: E2EPage;
     beforeEach(async () => {
       page = await newE2EPage({
-        html: `<calcite-sortable-list>
-        <div id="one"><calcite-handle></calcite-handle>1</div>
-        <div id="two"><calcite-handle></calcite-handle>2</div>
-        <div id="three"><calcite-handle></calcite-handle>3</div>
+        html: `<calcite-sortable-list draggable-selector=".calcite-sortable">
```

Comment:
This feature shouldn't replace the existing test. Can you add a separate for `draggable-selector` showcasing the difference?